setup.py

```diff
@@ -51,6 +51,8 @@
       "Programming Language :: Python :: 2.7",
       "Programming Language :: Python :: 3",
       "Programming Language :: Python :: 3.5",
+      "Programming Language :: Python :: 3.6",
```

Comment:
Did you run the tests with these Python versions?  When I cleaned this up originally I only listed versions I actually tested with a virtualenv and the listed python versions.